src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/FeatureSwitchManager.cs

```diff
@@ -591,6 +591,12 @@ private bool TryGetConstantArgument(MethodIL methodIL, byte[] body, OpcodeFlags[
                         {
                             return true;
                         }
+                        else if (method.IsIntrinsic && method.Name is "get_IsValueType"
+                            && method.OwningType is MetadataType { Name: "Type", Namespace: "System" }
```

Comment:
We check  mdType.Context.SystemModule in the existing case a few lines above. Do we need to do the same here?

It would be nice if both checks use the same style.

Reply:
I had that, then I learned `{..., Module: method.Context.SystemModule }` doesn't compile, became frustrated at how useless the pattern matching in C# is, and forgot. Fixed.